workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/container/OverlayDialogHolder.kt

```diff
@@ -79,7 +79,13 @@ public fun <OverlayT : Overlay> OverlayDialogHolder<OverlayT>.show(
   // Why is this an extension rather than part of the interface?
   // When wrapping, we need to prevent recursive calls from clobbering
   // `overlayOrNull` with the nested rendering type.
-  dialog.overlay = overlay
+
+  if (dialog.decorViewOrNull != null) {
+    // https://github.com/square/workflow-kotlin/issues/863
+    // This writes to the decor view, which is not safe to do before the
+    // dialog has been shown, because why would it be?
+    dialog.overlay = overlay
+  }
```

Comment:
Maybe a note that if the `decorViewOrNull` is null then the runner will cache this so it gets set after `Dialog.show`?